### ISSUE TITLE
Include C++ Compiler for Python in Easy-Setup

### ISSUE THREAD

**Issue by shadi1989** [PokemonGoMap/PokemonGo-Map#1185]
## Expected Behavior
...

## Current Behavior
Need to manually install the Microsoft Visual C++ Compiler for Python 2.7 when first using the maps.
https://www.microsoft.com/en-us/download/details.aspx?id=44266

## Possible Solution
Make an IF ELSE to ask a question if the user wants to install it or not.

## Steps to Reproduce (for bugs)
1. setting up the map on a fresh/new computer.

## Context
Easy win for first time users.

## Your Environment
* Version used: WINDOWS
* Environment name and version (e.g. Python 2.7): Python 2.7
* Operating System and version (desktop or mobile): Windows 7/8/10 only

**Comment by huanga** [comment]
You should not be rolling third party binaries in your package... Especially not OS + Platform specific binaries.

For easy setup, it might be easier to direct users to [Docker for Windows](https://docs.docker.com/docker-for-windows/), and have them follow the Docker guide instead.

**Comment by angrymoose** [comment]
There is absolutely no reason to include third party applications in the project. If one can't even manage to click a link and install the required packages they should not be running a website. The wiki may need updating but there is no reason some one who can read the wiki can't manage to click the download link & install an application. 

Furthermore as the project is licensed under the gpl they can not include third party files that are licensed under a closed source license.

TL;DR 

It violates the license of the project.
Os Specific binary file.
Won't Happen unless the maintainers are idiots.
